Makefile

```diff
@@ -3,13 +3,17 @@ all: dependencies test
 base_filename=kata
 
 dependencies:
-	which mit-scheme >/dev/null 2>&1 || brew install mit-scheme
```

Comment:
I don't think it's worth the readability of the Makefile, and I think it's important for beginners to see what's being run, but I can understand the other side of the argument